lib/slices/createCitySlice.js

```diff
@@ -0,0 +1,34 @@
+import { countries } from "../db";
+
+import { v4 as uuidv4 } from "uuid";
+export const createCitySlice = (set) => ({
+  countries,
+  addCity: (FormData) => {
+    const { city, country, startDate, endDate, note, hotels, places, food} = FormData;
```

Comment:
```suggestion
    const { city, countryID, startDate, endDate, note, hotels, places, food} = FormData;
```
If you're renaming country to countryID here (because I'm assuming that's what it is), you don't run into issues later on, what with the `_country` and everything.